### PR TITLE
Readme: Add Matrix badge and description

### PR DESCRIPTION
**Pull request by EwoutH**
Add a badge for the Matrix chat room and mention it in the readme

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -106,9 +107,12 @@ If you run into an issue, please file a `ticket`_ for us to discuss. If possible
 
 If you would like to add a feature, please reach out via `ticket`_ or the `dev email list`_ for discussion. A feature is most likely to be added if you build it!
 
+Mesa also has a `Matrix chat room`_ in which questions, issues and ideas can be (informatlly) discussed.
```
Comment (rht) [inline README.rst:110]: Typo `s/informatlly/informally/`.

File: README.rst [hunk 2]
```diff
@@ -106,9 +107,12 @@ If you run into an issue, please file a `ticket`_ for us to discuss. If possible
 
 If you would like to add a feature, please reach out via `ticket`_ or the `dev email list`_ for discussion. A feature is most likely to be added if you build it!
 
+Mesa also has a `Matrix chat room`_ in which questions, issues and ideas can be (informally) discussed.
```
Comment (rht) [inline README.rst:110]: Need to be Oxford comma: "questions, issues, and ideas".

File: README.rst [hunk 3]
```diff
@@ -5,12 +5,13 @@ Mesa: Agent-based modeling in Python 3+
...
 .. image:: https://codecov.io/gh/projectmesa/mesa/branch/main/graph/badge.svg
-  :target: https://codecov.io/gh/projectmesa/mesa
+        :target: https://codecov.io/gh/projectmesa/mesa
 
 .. image:: https://img.shields.io/badge/code%20style-black-000000.svg
-    :target: https://github.com/psf/black
+        :target: https://github.com/psf/black
 
-`Mesa`_ is an Apache2 licensed agent-based modeling (or ABM) framework in Python.
+.. image:: https://img.shields.io/matrix/projectmesa:matrix.org?label=Matrix%20chat&logo=Matrix
```
Comment (rht) [inline README.rst:13]: I think just labeling it "chat" is simpler. The logo already signifies that it is on Matrix.

**Comment by rht** [comment]
Other than the 2 comments (out of 3) I added, LGTM.

**Comment by rht** [comment]
One last nit: it should be `chat` instead of `Chat`. Notice that the other badge labels are in lowercase.

You can also see the cases in e.g. https://github.com/matrix-org/synapse, https://github.com/ethereum/go-ethereum, with some exceptions of course, https://github.com/servo/servo. But anyway, we should be consistent.

**Comment by EwoutH** [comment]
Thanks, done, rebased

**Comment by rht** [comment]
Merged, thank you!

**Pull request merged**